### DISCUSSION TITLE
I can't change the port from 8080 to 443

### DISCUSSION THREAD

**Discussion by JuanjoJG-WATA** [open-webui/open-webui#2879] [Q&A] [answered]
Hi!

I just updated the WebUI from v0.1.117. I tried to change the port in **start.sh** from 8080 to 443
![image](https://github.com/open-webui/open-webui/assets/165764759/9a181594-8ae8-4a50-aecb-886f43108260)
![image](https://github.com/open-webui/open-webui/assets/165764759/c5d65d8f-cb4e-43cf-9903-e2886ebe875a)

But when I run the start.sh, I get this error:
![image](https://github.com/open-webui/open-webui/assets/165764759/e168da80-eb59-4409-879e-2fb28489ca02)

When I try running it with sudo I get this:
![image](https://github.com/open-webui/open-webui/assets/165764759/73f9d391-fa8b-429e-b785-6c4be952f720)

**Comment by justinh-rahb** [comment]
`443` is a reserved port, you need `sudo` to bind to that. Besides, even if it wasn't.. this does not magically give you HTTPS if that was your assumption. You do not need to modify start.sh to change the port, just pass a `PORT` variable:
```bash
PORT=8081 bash start.sh
```

**Reply by JuanjoJG-WATA** [reply]
Yeah, yeah, I know, I was experimenting mostly. I created the **.crt** file and the **.key** file and I declared them in the start.sh. I did all of that because my server is not accessible from the internet and I can't use tools like Certbot, so a self-signed certificate was my only option (at least to the extent of my knowledge). 

In the old version I did it like that, just changing the start.sh file with my preferred port worked like a charm, but this one keeps saying permission denied.

Anyway, I'll try that and see if it works, thanks!

**Reply by JuanjoJG-WATA** [reply, marked as answer]
Fixed it, I did the `pip3 install -r requirements.txt -U` step as a regular user, not as sudo, so when I did, everything worked seamlessly.
